.github/workflows/kind_e2e.yaml

```diff
@@ -51,8 +51,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GOVER: 1.19.3
-      DAPR_RUNTIME_VERSION: 1.9.0
-      DAPR_DASHBOARD_VERSION: 0.11.0
+      DAPR_RUNTIME_PINNED_VERSION: 1.9.0
```

Comment:
Shall we update this to 1.9.4?

Reply:
Yes, I did this for testing. Will change it back to 1.9.4